assets/translations/data.fr-FR.json

```diff
@@ -72,18 +72,18 @@
                 },
                 "comptecommunal": "Compte communal",
                 "exceded":  {
-                    "maxNumber": "La limite de demande maximum est atteinte"
+                    "maxNumber": "La limite maximale de demandes est atteinte."
                 },
-                "printError": "Impossible d'imprimer les données du formulaire de demande",
-                "availableReqError": "Impossible de récupérer les demandes disponibles"
+                "printError": "Impossible d'imprimer les données du formulaire de demande.",
+                "availableReqError": "Impossible de récupérer les demandes disponibles."
             },
             "cadastre_tools" : "Outils Cadastre",
-            "create_line": "Sélectionne / Active / Désélectionne une ou plusieurs parcelles par une ligne brisée",
-            "create_point": "Sélectionne / Active / Désélectionne une parcelle par un simple clic",
-            "create_polygon": "Sélectionne / Active / Désélectionne une ou plusieurs parcelles à l’intérieur d’un polygone",
+            "create_point": "Sélectionner une parcelle par un simple clic",
+            "create_line": "Sélectionner des parcelles par une polyligne",
+            "create_polygon": "Sélectionnee des parcelles par un polygone",
```

Comment:
+1000 for the intent ! but typo on the last one..

Reply:
If correct, I can merge it

Reply:
the last one should be `Sélectionner` not `Sélectionnee`

Reply:
```suggestion
            "create_polygon": "Sélectionner des parcelles par un polygone",
```

Reply:
@landryb I'm going to merge this PR and fix this string in a separate update, to speed up the process